.github/workflows/continuous-integration.yml

```diff
@@ -180,6 +180,13 @@ jobs:
       - name: "Create cache directory for phpstan/phpstan"
         run: "mkdir -p .build/phpstan"
 
+      - name: "Cache cache directory for phpstan/phpstan"
+        uses: "actions/cache@v1"
+        with:
+          path: ".build/phpstan"
+          key: "php-${{ matrix.php-version }}-phpstan-${{ hashFiles('**/composer.lock') }}"
```

Comment:
libraries won't usually have a composer.lock committed.. shouldn't this build on composer.json?

Reply:
@staabm 

True for cases where it isn't checked in - but here it is!

🤓